content/plugin/framework/index.mdx

```diff
@@ -9,7 +9,7 @@ description: |-
 
 The plugin framework is a new way to develop Terraform plugins, providing improvements and new features from [Teraform Plugin SDKv2](/plugin/sdkv2/sdkv2-intro).
 
-~> **Important**: HashiCorp is currently developing providers against the framework, but it is still early in its development cycle. Its interfaces are subject to change and it may not yet support all of your desired features. It can also only build providers that require **Terraform v1.0.3 or later**.
+~> **Important**: HashiCorp is currently developing providers against the framework, but it is still early in its development cycle. Its interfaces are subject to change and it may not yet support all of your desired features. It can also only build providers that require **Terraform v1.0 or later**.
```

Comment:
```suggestion
~> **Important**: HashiCorp is currently developing providers against the framework, but it is still early in its development cycle. Its interfaces are subject to change and it may not yet support all of your desired features. It can also only build providers that require **Terraform CLI v1.0 or later**.
```